lean4-util.el

```diff
@@ -58,8 +58,8 @@ First try to find an executable named `lean4-executable-name' in
 
 (defun lean4-get-executable (exe-name)
   "Return fullpath of lean executable EXE-NAME."
-  (let ((lean4-bin-dir-name "bin"))
-    (f-full (f-join (lean4-get-rootdir) lean4-bin-dir-name exe-name))))
+  (let ((default-directory (lean4-get-rootdir)))
+    (expand-file-name exe-name (expand-file-name "bin"))))
```

Comment:
FYI, this could be `(file-name-concat (lean4-get-rootdir) "bin" exe-name)` after Emacs 28.1.